en/developer_guide/api.rst

```diff
@@ -0,0 +1,119 @@
+API
+===
+
+API access
+----------
+
+The connection to the API is based on the OAuth2 protocol.
+The OAuth2 protocol provides a client a secure access to an API on behalf of an user.
+
+A client is represented by a key/secret pair.
+In order to use the API, the client must request a token.
+
+The token is a string that refers to the client and optionally to the user. It is used to authenticate the client
+when he makes a request to the API.
+
+
+Client
+------
+
+The API clients can be managed in the administration section of the back office.
+
+The creation of a client is done through the add form.
+In the form, the key/secret pair is automatically generated by the server and
+the client can be specified like trusted.
+
+The trusted parameter allows you to prevent a connexion from a particular application to the api.
+
+Token
+-----
+
+Open Orchestra implements two ways to get a token. Both of those strategies are described in the OAuth2 protocol.
+
+Client credentials
+~~~~~~~~~~~~~~~~~~
+
+This strategy provides a token linked only to an API client.
+
+To obtain it, send the request:
+
+.. code-block::
+
+    /oauth/access_token?grant_type=client_credentials
+
+with the HTTP header:
+
+.. code-block::
+
+    Authorization: Basic ENCODED_PAIR
+
+Where ``ENCODED_PAIR`` is the string **key:secret** encoded in base64
+(obviously key and secret depend on the client).
+
+Resource owner password
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This strategy provides a token linked to an API client and a user.
+
+To obtain it, send the request:
+
+.. code-block::
+
+    /oauth/access_token?grant_type=password&username=USERNAME&password=PASSWORD
+
+Where USERNAME is user's name and PASSWORD is user's password.
+
+with the HTTP header:
+
+.. code-block::
+
+    Authorization: Basic ENCODED_PAIR
+
+Where ``ENCODED_PAIR`` is the string **key:secret** encoded in base64
+(obviously key and secret depend on the client).
+
+API usage
+---------
+
+To use the API, the parameter access_token=YOUR_TOKEN should be added in the query.
+
+Firewall configuration
+----------------------
+
+In the case where the API is used only, a stateless firewall must be defined in ``security.yml`` file:
```

Comment:
It could be. I see two possible construction : 
 - In the case where only the Api is used, 
 - When the Api is used on it's own,